axelor-human-resource/src/main/java/com/axelor/apps/hr/web/DPAEController.java

```diff
@@ -0,0 +1,90 @@
+package com.axelor.apps.hr.web;
+
+import com.axelor.apps.hr.db.DPAE;
+import com.axelor.apps.hr.db.repo.DPAERepository;
+import com.axelor.apps.hr.exception.IExceptionMessage;
+import com.axelor.apps.hr.service.app.AppHumanResourceService;
+import com.axelor.apps.hr.service.dpae.DPAEService;
+import com.axelor.common.ObjectUtils;
+import com.axelor.exception.AxelorException;
+import com.axelor.exception.ResponseMessageType;
+import com.axelor.exception.db.repo.TraceBackRepository;
+import com.axelor.exception.service.TraceBackService;
+import com.axelor.i18n.I18n;
+import com.axelor.inject.Beans;
+import com.axelor.meta.db.MetaFile;
+import com.axelor.meta.schema.actions.ActionView;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+import com.axelor.rpc.Context;
+import com.google.common.base.Function;
+import com.google.common.collect.Lists;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public class DPAEController {
+
+  @SuppressWarnings("unchecked")
+  public void send(ActionRequest request, ActionResponse response) throws AxelorException {
+    Context context = request.getContext();
+
+    try {
+      DPAEService dpaeService = Beans.get(DPAEService.class);
+      if (Beans.get(AppHumanResourceService.class).getAppEmployee().getTemplateDPAE() == null) {
+        throw new AxelorException(
+            DPAE.class,
+            TraceBackRepository.CATEGORY_MISSING_FIELD,
+            I18n.get(IExceptionMessage.DPAE_MISSING_TEMPLATE));
+      }
+      if (!ObjectUtils.isEmpty(request.getContext().get("_ids"))) {
```

Comment:
See pull #4231 this code could throw npe (even if it seems to be difficult to reproduce)